### DISCUSSION TITLE
v0.0.1 for betterNCM

### DISCUSSION THREAD

**Discussion by Entity-122425111303** [Entity-122425111303/UnblockCloudMusic#3] [Announcements]
### **v0.0.1 for betterNCM**
betterNCM方法解灰
The betterNCM method unlocks gray songs.

<hr /><em>This discussion was created from the release <a href='https://github.com/Entity-122425111303/UnblockCloudMusic/releases/tag/v0.0.1_for_betterNCM'>v0.0.1 for betterNCM</a>.</em>

**Comment by lkqstart** [comment]
hi 你有哪个BetterNCM ncm转mp3的插件吗

**Comment by SupremeShu** [comment]
请问解锁的音乐怎么更改音质呀，默认标准音质好糊
